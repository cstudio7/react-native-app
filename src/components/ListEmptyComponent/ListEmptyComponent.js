import React from 'react';
import { Text, View, StyleSheet } from 'react-native';
import { Colors } from '../../constants';

const ListEmptyComponent = ({ text }) => (
  <View style={styles.view}>
    <Text style={styles.text}>{text}</Text>
  </View>
);

const styles = StyleSheet.create({
  view: {
    flex: 1,
    flexDirection: 'column',
    justifyContent: 'center',
    alignItems: 'center',
    paddingTop: 50
  },
  text: {
    fontSize: 16,
    fontWeight: '500',
    color: Colors.secondary
  }
});

export default ListEmptyComponent;
