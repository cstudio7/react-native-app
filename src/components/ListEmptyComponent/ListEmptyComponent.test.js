import React from 'react';
import renderer from 'react-test-renderer';
import ListEmptyComponent from './ListEmptyComponent';

it('renders correctly', () => {
  const tree = renderer.create(<ListEmptyComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
