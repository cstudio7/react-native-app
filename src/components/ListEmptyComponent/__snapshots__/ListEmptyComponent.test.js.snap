// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly for Favorites screen 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "paddingTop": 50,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#938AA4",
        "fontSize": 16,
        "fontWeight": "500",
      }
    }
  >
    Вы пока ничего не добавили в избранное
  </Text>
</View>
`;

exports[`renders correctly for List screen 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "paddingTop": 50,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#938AA4",
        "fontSize": 16,
        "fontWeight": "500",
      }
    }
  >
    Загружается...
  </Text>
</View>
`;
